pipelines/deployer/deployer.yaml

```diff
@@ -790,3 +848,10 @@ jobs:
       action: apply
```

Comment:
Did you mean `destroy`, given this is the `destroy` job?

Reply:
I didn't write user-state, you want #323